content/platform/how-tos/minimise-costs.md

```diff
@@ -0,0 +1,35 @@
++++
+title = "Enable Preemptible Instances"
+weight = 4
+chapter = false
+pre = ""
++++
+
+{{% notice warning %}}
+Preemptible instances are not recommended for running workloads that are not fault tolerant. Instances will only run for 24 hours and will be turned off afterwards.
```

Comment:
why not use spot instances instead? Looks to me that spot vms are just the latest version of preemtible vms.

> Spot VMs are the latest version of preemptible VMs. New and existing preemptible VMs continue to be supported, and preemptible VMs use the same pricing model as Spot VMs. However, Spot VMs provide new features that preemptible VMs do not support. For example, preemptible VMs can only run for up to 24 hours at a time, but Spot VMs do not have a maximum runtime unless you [limit the runtime](https://cloud.google.com/compute/docs/instances/limit-vm-runtime).

Reply:
feels to me that we should default to spot and only if client is unable to use it (not sure what could be the reason for it yet) we can fallback to preemtible